Artifacts/windows-deploymentgroup/InstallAndConfigureDeploymentGroupAgent.ps1

```diff
@@ -144,6 +144,76 @@ function Configure-DeploymentGroupAgent
 
     if ($runAsAutoLogon)
     {
+        # Create a PS session for the user to trigger the creation of the registry entries required for autologon
+        if ($windowsLogonAccount.Split("\").Count -eq 2)
+        {
+            $domain = $windowsLogonAccount.Split("\")[0]
+        }
+        else
+        {
+            $domain = Hostname
+        }
+        $computerName = "localhost"
```

Comment:
Will `localhost` always work? Do you need to consider something like `"$($Env:ComputerName)"`?

Reply:
It has to be localhost. It doesn't work with $($Env:ComputerName) as this login has to bypass the NIC and go via loopback since azure VMs don't support pssessions via http. (only https supported)